components/builder-api/src/config.rs

```diff
@@ -64,6 +66,7 @@ impl Default for Config {
             github_client_secret: DEV_GITHUB_CLIENT_SECRET.to_string(),
             ui_root: None,
             events_enabled: false, // TODO: change to default to true later
+            log_dir: "/tmp".to_string(),
```

Comment:
we probably want to abstract temp directory since it would be the temp environment variable on windows.